src/app/common/components/modals/dialog-create-or-update-wallet/wallet-form-modal.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA }  from '@angular/material/dialog';
+import { IWalletModalData } from 'src/app/domains/wallets/domains.wallets.types';
+
+@Component({
+  selector: 'app-wallet-form-modal',
+  templateUrl: './wallet-form-modal.component.html',
+  styleUrls: ['./wallet-form-modal.component.scss']
+})
+export class WalletFormModalComponent {
```

Comment:
If you are not 100% sure that this component will be used in more than one place or will be a generic component used in one place but displayed everywhere (like navbar component) - I would place it inside `pages/wallets/my-wallets` and name properly :)
We can always move it here if needed.